integration-tests/terraform/testbuild/helm/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
```

Comment:
Where are we using all these helm folder files? Why do we need all this files here?

Reply:
I am using the exsiting Helm now.